webtop/__init__.py

```diff
@@ -144,6 +182,16 @@ def render_stats(stats: dict, _format: str) -> str:
     args = parse_args()
     assert are_args_valid(args)
 
+    custom_resolution = {}
```

Comment:
`custom_resolution` is only used if args.resolve is defined, and can be moved below that if

Reply:
I'll move this all the way in and replace the key assignment with a dictionary literal. MAXIMUM BREVITY.